File: Execício 5 - Revisão Prática/relacionais/relacionais.java
import java.util.Scanner;

public class relacionais {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Digite dois números inteiros \n");

        if (!sc.hasNextInt()) { System.out.println("Entrada inválida."); return; }
        int a = sc.nextInt();
        if (!sc.hasNextInt()) { System.out.println("Entrada inválida."); return; }
        int b = sc.nextInt();

        System.out.println("a > b ? " + (a > b));
        System.out.println("a == b ? " + (a == b));
        System.out.println("Ambos pares? " + (a % 2 == 0 && b % 2 == 0));

        sc.close();
    }
}

File: Execício 5 - Revisão Prática/relacionais/relacionais.c
#include <stdio.h>

int main() {
    int a, b;
    if (scanf("%d %d", &a, &b) != 2) {
        printf("Entrada inválida.\n");
        return 1;
    }
    
    printf("Digite dois números inteiros");
    printf("\n\na > b ? %s\n", (a > b) ? "true" : "false");
    printf("a < b ? %s\n", (a < b) ? "true" : "false");
    printf("a == b ? %s\n", (a == b) ? "true" : "false");
    printf("a != b ? %s\n", (a != b) ? "true" : "false");
    printf("a >= b ? %s\n", (a >= b) ? "true" : "false");
    printf("a <= b ? %s\n", (a <= b) ? "true" : "false");

    
    printf("Ambos pares? %s\n", ( (a % 2 == 0) && (b % 2 == 0) ) ? "true" : "false");

    
    }

    
